packages/core/src/components/Spinner/Spinner.scss

```diff
@@ -23,9 +23,9 @@ Spinners signify that the application is waiting for an asynchronous operation t
 @status alpha
 
 Markup:
-<span class="ds-c-spinner ds-c-spinner--small"></span>
-<span class="ds-c-spinner"></span>
-<span class="ds-c-spinner ds-c-spinner--big"></span>
+<span class="ds-c-spinner ds-c-spinner--small" aria-valuetext="Loading" role="progressbar"></span>
```

Comment:
I see these are documented in the React component, but we should probably expand on these in the "Accessibility" section of the Guidance section. For example, according to [MDN](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_aria-valuetext_attribute) `aria-valuetext` should only be set when the rendered value can't be accurately represented as a number.